packages/eslint-plugin/src/rules/prefer-find.ts

```diff
@@ -153,7 +153,14 @@ export default createRule({
      * @see https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/at#parameters
      */
     function isTreatedAsZeroByArrayAt(value: unknown): boolean {
-      const asNumber = Number(value);
+      let asNumber: number;
+
+      try {
```

Comment:
[Refactor] I'm not a big fan of `try`/`catch` as a form of program control flow. It somewhat hides the intent of the code. Thrown errors are generally a sign of something truly breaking - IMO it's better to be intentional around code always working as expected.

Instead of generally catching all errors, how about checking if `typeof value` is `'symbol'`?

Reply:
yep, I was 50/50 on which way to go on this. Changed!